core/perf_test/PerfTest_ViewCopy.cpp

```diff
@@ -42,16 +42,25 @@
 //@HEADER
 */
 
-#include <benchmark/benchmark.h>
+#include <PerfTest_ViewCopy.hpp>
 
-void SomeFunction() { return; }
+namespace Test {
 
-static void BM_SomeFunction(benchmark::State& state) {
-  // Perform setup here
-  for (auto _ : state) {
-    // This code gets timed
-    SomeFunction();
-  }
+/**
+ * \brief Mark the label as a figure of merit.
+ */
+std::string benchmark_fom(const std::string& label) { return "FOM: " + label; }
+
+void report_results(benchmark::State& state, double time) {
+  state.SetIterationTime(time);
+
+  const auto N8        = std::pow(state.range(0), 8);
+  const auto size      = N8 * 8 / 1024 / 1024;
+  state.counters["MB"] = benchmark::Counter(size, benchmark::Counter::kDefaults,
+                                            benchmark::Counter::OneK::kIs1024);
+  state.counters[benchmark_fom("GB/s")] =
+      benchmark::Counter(2 * size / 1024 / time, benchmark::Counter::kDefaults,
+                         benchmark::Counter::OneK::kIs1024);
```

Comment:
Could you please provide some more comments on the magic constants here?

Reply:
if he reveals them, they would not be magic anymore :)

Reply:
(possibly something like `const auto size_mb = view.size() * sizeof(double) / 1024 / 1024;`?)